databases/week3/Queries.sql

```diff
@@ -0,0 +1,158 @@
+-- Meal
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date)
+values ('Homemade Pasta', 'Gnocchi with pesto, ragu or mushroom sauce', 'Valby', '2021-10-14',10, 10.5, now()),
+       ('Homemade Gnocchi','Gnocchi with pesto, ragu or mushroom sauce', 'Amager', '2021-04-12 15:17:08',10, 200, '2021-04-13 17:31:16'),
+	   ('Homemade spagetti', 'Gnocchi with pesto, ragu or mushroom sauce', 'Valby', '2021-10-14',10, 10.5, now()),
+	   ('Authentic Tortellini', 'This simple soup with cheese tortellini, Italian sausage, and chopped fresh spinach is quick-and-easy to prepare.', 'Valby', '2021-10-14',10, 100, now());
+
+#1 Get all meals       
+SELECT *
+FROM meal;
+
+# Add a new meal
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date)
+values ('Authentic Tortellini Soup', 'This simple soup with cheese tortellini, Italian sausage, and chopped fresh spinach is quick-and-easy to prepare.', 'Valby', '2021-10-14',10, 100, now());
+
+# Get a meal with any id, fx 1
+SELECT *
+FROM meal
+WHERE title='Homemade Pasta';
+
+# Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE meal
+SET description = 'This is a light and garlicky spaghetti sauce.', location= 'Milan', price=125
+WHERE title LIKE '%spagetti%';
+
+# Delete a meal with any id, fx 1
+DELETE FROM meal
+WHERE price = 100;
+
+-- Reservation
+# Get all reservations
+SELECT *
+FROM reservation;
+
+# Add a new reservation
```

Comment:
I know these queries work, but they are not easy to understand at all, so can you do it the old simple way please?

For the first one it would look like this:
```
INSERT INTO reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) 
VALUES (2, 1, now(), 71708085, 'Aarika Ellingworth', 'aellingworth0@harvard.edu' );
```